examples/daymet-hi-annual.json

```diff
@@ -0,0 +1,186 @@
+    "license": "proprietary",
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -160.3056,
+                    17.9539,
+                    -154.772,
+                    23.5186
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "1980-01-01T12:00:00Z",
+                    "2019-12-31T12:00:00Z"
+                ]
+            ]
+        }
+    },
+    "links": [],
+    "assets": {
+        "data": {
+            "href": "az://daymet-zarr/annual/hi.zarr",
+            "title": "Zarr store",
+            "type": "application/fsspec/zarr",
+            "description": "Root URL of the Zarr store",
+            "roles": [
+                "data"
+            ]
+        }
+    },
+    "cube:dimensions": {
+        "time": {
+            "type": "temporal",
+            "extent": [
+                "1980:00:00T00:00:00Z",
+                "2020:00:00T00:00:00Z"
+            ],
+            "description": "Annual",
+            "step": "P1Y"
+        },
+        "x": {
+            "type": "spatial",
+            "axis": "x",
+            "description": "North American Lambert Conformal Conic, 1KM grid.",
+            "extent": [
+                -622000,
+                -5519250
+            ],
+            "step": 1000,
+            "reference_system": "PROJCRS[\"undefined\",BASEGEOGCRS[\"undefined\",DATUM[\"undefined\",ELLIPSOID[\"undefined\",6378137,298.257223563,LENGTHUNIT[\"metre\",1,ID[\"EPSG\",9001]]]],PRIMEM[\"Greenwich\",0,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8901]]],CONVERSION[\"unknown\",METHOD[\"Lambert Conic Conformal (2SP)\",ID[\"EPSG\",9802]],PARAMETER[\"Latitude of 1st standard parallel\",25,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8823]],PARAMETER[\"Latitude of 2nd standard parallel\",60,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8824]],PARAMETER[\"Latitude of false origin\",42.5,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8821]],PARAMETER[\"Longitude of false origin\",-100,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8822]],PARAMETER[\"Easting at false origin\",0,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8826]],PARAMETER[\"Northing at false origin\",0,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8827]]],CS[Cartesian,2],AXIS[\"(E)\",east,ORDER[1],LENGTHUNIT[\"metre\",1,ID[\"EPSG\",9001]]],AXIS[\"(N)\",north,ORDER[2],LENGTHUNIT[\"metre\",1,ID[\"EPSG\",9001]]]]"
```

Comment:
This is ugly, but Daymet uses its own projection (slightly different from http://epsg.io/102009).

I can remove this / lie about what the projection is if you want to make this simpler.

Reply:
That's how it is right now. In a future version of this extension we may consider using proj:wkt/projjson/epsg fields, but for now this is fine.